src/main/java/com/bulkbuy/enterprise/dao/OrderDAO.java

```diff
@@ -1,4 +1,26 @@
 package com.bulkbuy.enterprise.dao;
 
-public class OrderDAO {
+import com.bulkbuy.enterprise.dto.Order;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class OrderDAO implements IOrderDAO{
+    List<Order> allOrders = new ArrayList<>();
+
+    @Override
+    public void create(Order order) {
+        try {
+            allOrders.add(order);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public List<Order> listAll() {
+        return allOrders;
+    }
```

Comment:
Created classes for adding a new order and returning a list of orders in the DAO.